lib/dl_connector_bundle_chs3/dl_connector_bundle_chs3/locales/en/LC_MESSAGES/dl_connector_bundle_chs3.po

```diff
@@ -12,7 +12,7 @@ msgstr ""
 msgid "label_connector-gsheets_v2"
 msgstr "Google Sheets"
 
-msgid "label_connector-yadocs"
+msgid "label_connector-docs"
```

Comment:
Regenerated with tasks?

Reply:
This question is the only thing stopping me from approving this pr
Others can be addressed separately